CODEOWNERS

```diff
@@ -32,8 +32,7 @@ cmake/ @tt-rkim @vtangTT @TT-billteng
 build_metal.sh @tt-rkim @vtangTT @TT-billteng
 
 Makefile @tt-rkim
-/CMakeLists.txt @tt-rkim @vtangTT @TT-billteng
-tt_metal/CMakeLists.txt @tt-rkim @vtangTT @TT-billteng
+**/CMakeLists.txt @tt-rkim @vtangTT @TT-billteng @blozano-tt
```

Comment:
Do we want to control all CMake files that's not the root one as well?

Reply:
I think we should. They are all related.

Reply:
Is there a way we can be co-owners along with the developers?

(I think that’s how it is in this PR).

I’d like to help review changes to any CMakeLists.txt

Reply:
No, subsequent lines erase any previously-declared ones.

So let's say we want devA and devB to help us own `tt_metal/CMakeLists.txt`. Then we need:

```
**/CMakeLists.txt @tt-rkim @TT-billteng @blozano-tt

...

tt_metal/CMakeLists.txt @tt-rkim @TT-billteng @blozano-tt @devA @devB
```

BTW, I think we should take out @vtangTT from this since he's part time and would be spamming him a lot. Maybe we should leave him on for top level `/CMakeLists.txt`.

Reply:
I have never worked with the CODEOWNERS file before, but wouldn't this later declaration remove us from being code owners of tt_metal/CMakeLists.txt then?

https://github.com/tenstorrent/tt-metal/blob/943e140ee20b2db8181cae4f7cb3871fe524939d/CODEOWNERS#L53

Reply:
Revised my request.